documentation/modules/ROOT/pages/connectors/postgresql.adoc

```diff
@@ -2947,6 +2947,24 @@ For example: `CREATE PUBLICATION <publication_name> FOR TABLE <tbl1, tbl2, tbl3>
 If the publication exists, the connector updates the publication for tables that match the current filter configuration.
 For example: `ALTER PUBLICATION <publication_name> SET TABLE <tbl1, tbl2, tbl3>`.
 
+|[[postgresql-replica-autoset-type]]<<postgresql-replica-autoset-type, `+replica.identity.autoset.values+`>>
+|_empty string_
+|Applies only when streaming changes by using link:https://www.postgresql.org/docs/current/sql-createpublication.html[the `pgoutput` plug-in].
```

Comment:
Why it should be alloed only for `pgoutput`?

Reply:
I don't actually have a reason to use the Replica Identity only for `pgoutput`. So, let's do it for `decoderbufs` as well.